kuma/users/tests/test_views.py

```diff
@@ -1262,6 +1262,11 @@ def test_matching_accounts(self):
         assert response.status_code == 200
         assert_no_cache_header(response)
         assert not response.context['matching_accounts']
+        # The template is tested here instead of test_templates.py because
+        # test setup is so painful for login tests.
+        parsed = pq(response.content)
+        li_exists = parsed.find('ul.choices li.exists')
+        assert len(li_exists) == 0
```

Comment:
```suggestion
        assert not li_exists, li_exists
```

The `li_exists' (from the `.find()`) becomes an empty list if nothing was found.